Cargo.toml

```diff
@@ -18,6 +18,6 @@ strip = true
 # We enable some basic optimization to prevent stack overflows during testing (flybywiresim/a32nx#7631)
 # However, if this interferes with your ability to debug locally you can disable it here
 opt-level=1
-lto = false
+lto = true
```

Comment:
Is  this change necessary? Maybe you left it in by accident?

Reply:
I had a rust compiler crash that this solved. But possibly this has been resolved by then, will try.

Reply:
Still crashing for unknown reason when using lto=false :(